src/Discord.Net.WebSocket/Entities/Channels/SocketChannel.cs

```diff
@@ -41,7 +41,7 @@ internal static ISocketPrivateChannel CreatePrivate(DiscordSocketClient discord,
 
         //IChannel
         string IChannel.Name => null;
-        bool IChannel.IsNsfw => ChannelHelper.IsNsfw(this);
+        bool IChannel.IsNsfw => false;
```

Comment:
Again, it doesn't really make sense that we're only exposing IsNsfw on IChannel.